pipelines/build/common/build_base_file.groovy

```diff
@@ -181,7 +181,7 @@ class Builder implements Serializable {
             ENABLE_TESTDYNAMICPARALLEL: enableTestDynamicParallel,
             ENABLE_INSTALLERS: enableInstallers,
             ENABLE_SIGNER: enableSigner,
-            CLEAN_WORKSPACE: cleanWorkspace,
+            CLEAN_WORKSPACE_BEFORE_BUILD: cleanWorkspaceBeforeBuild,
```

Comment:
```suggestion
            CLEAN_WORKSPACE_BEFORE: cleanWorkspaceBeforeBuild,
```

Reply:
Suggested change so that it's the same as the AFTER naming convention.  If accepted you'll need to change everywhere